### DISCUSSION TITLE
How to fine tuning using jsut copas tacotron2 model

### DISCUSSION THREAD

**Discussion by asao-kanta** [espnet/espnet#5218] [Q&A] [answered]
**Describe the issue**

@kan-bayashi 

I am currently looking to do a fine-tuinig of fastspeech2 using the jsut corpus on my own Japanese dataset.
However, I am having trouble completing the fine-tuning of tacotoron2.

There is no tuning-tacotron2.yaml file like the one in the jvs corpus, but is there a detailed reference as to which file to use and where to reference the data?

i read https://github.com/espnet/espnet/blob/master/egs2/jvs/tts1/conf/tuning/finetune_tacotron2.yaml and https://kan-bayashi.github.io/asj-espnet2-tutorial/, but i could not understand.

I am a beginner and this is a rudimentary question, but I would appreciate an answer
thank you

**Comment by kan-bayashi** [comment, marked as answer]
サンプリングレート等が同じであれば、JVSのものをコピーして使えます！
